layers/auto-completion/packages.el

```diff
@@ -269,7 +269,7 @@
         "Call `yas-expand' and switch to `insert state'"
         (interactive)
         (call-interactively 'aya-expand)
-        (evil-insert-state))
+        (unless holy-mode (evil-insert-state)))
```

Comment:
@justbur It's the hybrid state that was annoying. It's a weird UX to be put in hybrid mode after some command when you are using the emacs editing style.